projects/material-extended/mde/src/lib/popover/popover-trigger.ts

```diff
@@ -230,26 +240,19 @@ export class MdePopoverTrigger implements AfterViewInit, OnDestroy { // tslint:d
     openPopover(): void {
         if (!this._popoverOpen && !this._halt) {
             this._createOverlay().attach(this._portal);
-
-            /** Only subscribe to backdrop if trigger event is click */
-            if (this.triggerEvent === 'click' && this.backdropCloseOnClick === true) {
-              this._subscribeToBackdrop();
-            }
+    
+            this._subscribeToBackdrop();
+            this._subscribeToDetachments();
 
             this._initPopover();
         }
     }
 
     /** Closes the popover. */
     closePopover(): void {
+      console.log('close');
```

Comment:
Forgotten console.log

Reply:
@quanterion

Thanks. This is still a WIP so before I merge into master I will remove that. 

There's just a bug with the events that I found when creating this feature. I'm not sure if I've fixed that bug yet because I got dragged back into other projects.

Reply:
My mistake this is in master and I haven't released an updated npm version yet. It's possible I merged too soon. I will have to review everything before the next release.

Reply:
@joejordanbrown Are there any plans to release this?